guides/micronaut-oauth2-auth0/micronaut-oauth2-auth0.adoc

```diff
@@ -0,0 +1,123 @@
+include::{commondir}/common-header.adoc[]
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+== OAuth 2.0
+
+To provide authentication, sign in to your https://auth0.com[Auth0] account.
+
+Create an Application
+
+image::auth0-create-application.png[]
+
+Fill the Application URIs:
+
+image::auth0-application-uris.png[]
+
+- Enter `http://localhost:8080/oauth/callback/auth0` as callback url.
+
+- Enter `http://localhost:8080/logout` as allowed logout url.
+
+- Enter `http://localhost:8080` as allowed web origins.
+
+Once you have an https domain (e.g. `https://myapp.org`) enter it as Application Login URI: `https://myapp.org/oauth/login/auth0`.
```

Comment:
Use backslash to disable auto-rendering of these URLs, and capitalize URL and HTTPS.

```suggestion
- Enter `\http://localhost:8080/oauth/callback/auth0` as callback URL.

- Enter `\http://localhost:8080/logout` as allowed logout URL.

- Enter `\http://localhost:8080` as allowed web origins.

Once you have an HTTPS domain (e.g. `\https://myapp.org`) enter it as Application Login URI: `\https://myapp.org/oauth/login/auth0`.
```

Reply:
thanks. I did not know about auto-rendering disabling with asciidoc